api/pkg/apis/v1alpha1/providers/stage/materialize/materialize.go

```diff
@@ -283,32 +257,6 @@ func (i *MaterializeStageProvider) Process(ctx context.Context, mgrContext conte
 						return outputs, false, v1alpha2.NewCOAError(nil, fmt.Sprintf("Empty target name: catalog - %s", name), v1alpha2.BadRequest)
```

Comment:
Why we keed createtargetcontainer instead of createtarget? Again, it is better not assume the error is "Empty target name".